### PR TITLE
v6.0.0-beta.2

### PR DESCRIPTION
**Pull request by flaviendelangle**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,57 @@
...
+- [docs] Fix a typo in data grid layout page (#7113) @sfbaker7
+- [docs] Fix require context path to avoid duplicate key creation (#7781) @LukasTy
+- [docs] Polish pickers migration docs (#7737) @LukasTy
+- [docs] Rename `next` translation docs and remove duplicates with `-next` (#7729) @LukasTy
+
+### Core
+
+- [core] Bump `loader-utils` (#7735) @LukasTy
+- [core] Fix Next.js warning (#7754) @oliviertassinari
+- [core] Remove `rifm` dependency (#7785) @alexfauquette
```
[the remark this answers is not in the document]
Comment (flaviendelangle) [reply CHANGELOG.md:54]: That's probably better indeed

File: CHANGELOG.md [hunk 2]
```diff
@@ -2021,6 +2073,32 @@ You can find more information about the new api, including how to set those tran
 - [test] Skip tests for column pinning and dynamic row height (#5997) @m4theushw
 - [website] Improve security header @oliviertassinari
 
+## v5.17.22
```
Comment (LukasTy) [inline CHANGELOG.md:2076]: We usually do not include `v` in these headers.
```suggestion
## 5.17.22
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,58 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v6.0.0-beta.2
```
Comment (LukasTy) [inline CHANGELOG.md:6]:
```suggestion
## 6.0.0-beta.2
```

File: yarn.lock [hunk 4]
```diff
@@ -2506,6 +2506,47 @@
     prop-types "^15.8.1"
     react-is "^18.2.0"
 
+"@mui/x-data-grid-premium@6.0.0-beta.1":
```
Comment (LukasTy) [inline yarn.lock:2509]: Shouldn't the `x-data-grid-generator` also be bumped to avoid these changes? 🤔

Comment (flaviendelangle) [reply yarn.lock]: Indeed, done :+1:

**Review by LukasTy — APPROVED** [review]

**Review by MBilalShafi — APPROVED** [review]

**Pull request merged**